bonfire/bonfire.py

```diff
@@ -815,14 +832,14 @@ def _describe_namespace(namespace):
     click.echo(describe_namespace(namespace))
 
 
-def _get_apps_config(source, target_env, ref_env, local_config_path, local_config_method):
+def _get_apps_config(source, target_env, ref_env, local_config_path, local_config_method, prefer):
```

Comment:
maybe changed `prefer` to `preferred_params` here to retain consistency? Also, `prefer` gives me "this is probably a boolean" vibes whereas `preferred_params` sounds more like some kind of collection or dictionary.

Reply:
Good idea, will implement